ocp_resources/controller_revision.py

```diff
@@ -1,42 +1,36 @@
-# https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/controller-revision-v1/
-from ocp_resources.constants import TIMEOUT_4MINUTES
 from ocp_resources.resource import NamespacedResource
 
 
 class ControllerRevision(NamespacedResource):
+    """
+    https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/controller-revision-v1/
+    """
+
     api_group = NamespacedResource.ApiGroup.APPS
 
     def __init__(
         self,
-        name=None,
-        namespace=None,
-        client=None,
-        teardown=True,
-        timeout=TIMEOUT_4MINUTES,
-        privileged_client=None,
-        yaml_file=None,
-        delete_timeout=TIMEOUT_4MINUTES,
         owner_references=None,
         revision_object=None,
         **kwargs,
     ):
-        super().__init__(
-            name=name,
-            namespace=namespace,
-            client=client,
-            teardown=teardown,
-            timeout=timeout,
-            privileged_client=privileged_client,
-            yaml_file=yaml_file,
-            delete_timeout=delete_timeout,
-            **kwargs,
-        )
+        """
+        Args:
+            owner_references (list): List of objects depended on this object.
+            revision_object (int): indicates the revision of the state represented by Data.
```

Comment:
@rnetser @myakove  I cannot verify this PR,
The changes I added are verified but the API docs mention the revision_object type is int, while line 39 is trying to interact with `revision_object.res` which doesn't match. What do you think?

[See](https://github.com/RedHatQE/openshift-python-wrapper/pull/1444/files#diff-3127d82cecc19690fb34f0c1c7bb69f195b28ee086bcbd3bc09e1a0e473aeed3R39)

Reply:
@oharan2 update `revision_object` description; it is not an int.
 `revision_object` is object spec.

Reply:
I thought the `revision_object` and the revision parameter is referring the same, will update

Reply:
@rnetser Do I need to validate the Int64 (as recently committed)? 
many cases might pass a regular int and I suggested to maintain this requirement.